code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -472,13 +472,20 @@
 	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
 	toxpwr = 0.5
 
+/datum/reagent/toxin/spidervenom/on_mob_metabolize(mob/living/L)
+	if(SEND_SIGNAL(L, COMSIG_HAS_NANITES))
+		for(var/datum/component/nanites/N in L.datum_components)
+			for(var/X in N.programs)
+				var/datum/nanite_program/NP = X
+				NP.software_error(5) //all programs are overwritten with /glitch which does no harm, but gradually drains nanites. 
```

Comment:
will this continually apply every metabolize tick?

Reply:
I swore I moved this to the other proc in my last commit, I guess not :/
Thanks for the catch.

Reply:
I **DID** move this to the correct proc in my last commit now that I have the code open to "fix" this.

`on_mob_metabolize()` is called only once when a reagent first enters the system